tests/DoctrineModuleTest/Service/Authentication/StorageFactoryTest.php

```diff
@@ -67,8 +70,46 @@ public function testCanInstantiateStorageFromServiceLocator(): void
             ->will($this->returnValue($storage));
 
         $this->assertInstanceOf(
-            'DoctrineModule\Authentication\Storage\ObjectRepository',
+            ObjectRepository::class,
             $factory->createService($serviceManager)
         );
     }
+
+    public function testCanInstantiateCustomStorage(): void
+    {
+        $factory        = new StorageFactory('customFactory');
+        $serviceManager = $this->createMock(ServiceManager::class);
+        $config         = [
+            'doctrine' => [
+                'authentication' => [
+                    'customFactory' => [
+                        'sessionContainer' => 'customContainer',
+                        'sessionMember' => 'customMember',
+                        'storage' => 'custom_storage',
+                    ],
+                ],
+            ],
+        ];
+
+        $serviceManager
+            ->expects($this->at(0))
```

Comment:
`at` is deprecated in recent versions of PHPUnit, you should avoid it or will mean more work when migrating.

If you mean to check that the service manager is called, use

```suggestion
            ->expects($this->once())
```

If you mean to just set up the service manager, then this is a stub, and you can drop the line entirely (and use `createStub` over `createMock`.

Another solution could be to use a real service manager instead of a mock.

Reply:
It is a mock, not a stub.  Changed to use `onConsecutiveCalls` and removed `->at()` calls.